features/features.go

```diff
@@ -548,4 +548,11 @@ var (
 							productScope(ocpSpecific).
 							enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 							mustRegister()
+
+	FeatureGateMonitoring = newFeatureGate("Monitoring").
```

Comment:
(commented off-line already)
```suggestion
	FeatureGateMonitoring = newFeatureGate("ClusterMonitoringConfig").
```